### PR TITLE
Replaced description by data format of SQLite parser plugins #3087

### PR DESCRIPTION
**Pull request by joachimmetz**
Replaced description by data format of SQLite parser plugins #3087

### CODE REVIEW

**Comment by rgayon** [comment]
Crap, didn't want to approve

File: plaso/parsers/sqlite_plugins/hangouts_messages.py [hunk 1]
```diff
@@ -43,10 +34,20 @@ def __init__(self):
 
 
 class HangoutsMessagePlugin(interface.SQLitePlugin):
-  """Parser for Google Hangouts databases."""
+  """SQLite parser plugin for Google Hangouts conversations database files.
+
+  The oogle Hangouts conversations database file is typically stored in:
```
[the remark this answers is not in the document]
Comment (joachimmetz) [reply plaso/parsers/sqlite_plugins/hangouts_messages.py:39]: done

File: plaso/parsers/sqlite_plugins/interface.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-"""The SQLite parser plugin interface."""
+"""Interface for SQLite database file parser plugins."""
```
[the remark this answers is not in the document]
Comment (joachimmetz) [reply plaso/parsers/sqlite_plugins/interface.py:2]: done

File: plaso/parsers/sqlite_plugins/mac_notificationcenter.py [hunk 3]
```diff
@@ -57,11 +43,22 @@ def __init__(self):
 
 
 class MacNotificationCenterPlugin(interface.SQLitePlugin):
-  """Parser for MacOS Notification Center SQLite database files."""
+  """SQLite parser plugin for MacOS Notification Center database files.
+
+  The  MacOS Notification Center database file is typically stored in:
+  /private/var/folders/<W><d>/../0/com.apple.notificationcenter/db2/db
```
[the remark this answers is not in the document]
Comment (joachimmetz) [reply plaso/parsers/sqlite_plugins/mac_notificationcenter.py:49]: not sure what you're referring to? are you referring to `<W><d>` e.g. `/var/folders/_d/pg2g__g17nzfwrv64j8ddvn80000gn/0//com.apple.notificationcenter/db`

**Comment by joachimmetz** [comment]
@rgayon PTAL

File: plaso/parsers/sqlite_plugins/mac_notificationcenter.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (rgayon) [reply plaso/parsers/sqlite_plugins/mac_notificationcenter.py:49]: Thank you github for not displaying the ` <W><d>`, but yeah that.
If this is expected the okay

**Pull request merged**